data/json/overmap_specials.json

```diff
@@ -480,14 +480,14 @@
         "type" : "overmap_special",
         "id" : "Apartments_Mod",
         "overmaps" : [
-            { "point":[0,0,0], "overmap": "apartments_mod_tower_1"},
-            { "point":[1,0,0], "overmap": "apartments_mod_tower_1_entrance"},
-            { "point":[0,1,0], "overmap": "apartments_mod_tower_1"},
-            { "point":[1,1,0], "overmap": "apartments_mod_tower_1"}
+            { "point": [ 0, 0, 0 ], "overmap": "apartments_mod_tower_NW_north" },
+            { "point": [ 1, 0, 0 ], "overmap": "apartments_mod_tower_NE_north" },
+            { "point": [ 0, 1, 0 ], "overmap": "apartments_mod_tower_SW_north", "connect": "road" },
```

Comment:
`"connect": "road"` isn't supported anymore. Only `"connections" : [ ... ]` is used.